VKApp/VKApp/TabBar/Thrird/AuthorTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+// AuthorTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Автора поста
+final class AuthorTableViewCell: UITableViewCell {
+    // MARK: - IBOutlets
+
+    @IBOutlet private var profilePhotoImageView: UIImageView!
+    @IBOutlet private var userNameLabel: UILabel!
+    @IBOutlet private var authorView: UIView!
+    @IBOutlet private var viewsCountLabel: UILabel!
+
+    // MARK: - Public methods
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        configPostView()
+    }
+
+    func configureUser(_ post: PostItem, networkService: NetworkService) {
+        viewsCountLabel.text = String(post.count)
+//        userNameLabel.text = "\(item.firstName) \(item.lastName)"
```

Comment:
```suggestion
//        userNameLabel.text = "\(item.firstName) \(item.lastName)"
```
незя закоменченый код оставлять